sklearn/preprocessing/data.py

```diff
@@ -54,41 +58,82 @@
 )
 
 
-def _mean_and_std(X, axis=0, with_mean=True, with_std=True):
-    """Compute mean and std deviation for centering, scaling.
+def _incremental_mean_and_var(new_X, axis=0, last_mean=.0, last_var=None,
```

Comment:
This is indeed a YAGNI. Let's remove it to simplify the code. It's always easy to re-add it later when we actually need that parameters.